### PR TITLE
Week 4: Project Portfolio

### PR DESCRIPTION
**Pull request by EmmaEngvall**
I have not completed this week's project on time. I will continue working on it in the coming weeks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property ="og:title" content="Emmas Portfolio"/>
+    <meta property ="og:description" content="The portfolio page for Emma Engvall aspiring frontend developer with a background as Business Analyst."/>
+    <meta property ="og:image" content="./images/logo.jpg"/>
```
Comment (JosefinRobertsson) [inline index.html:9]: Good relevant og:tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    <link rel="stylesheet" href="./style.css">
+    <title>Emma Engvall Portfolio</title>
+</head>
+<body>
+    <section class="logo">
+        <img src="./images/logo.jpg" alt="Portfolio logo" width="700">
+    </section>
+    <main>
+    <section class="header">
+        <img src="./images/img_profile.png" alt="Image of Emma Engvall" class="profile" width="164">
```
Comment (JosefinRobertsson) [inline index.html]: I think you meant to set the width in your CSS document.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        <h1>Emma Engvall</h1>
+        <h2 class="Title">Frontend developer</h2>
+        <h3 class="Work">+ business analyst</h3> 
+        
+        <p> 
+            My name is Emma and I am an aspiring frontend developer.
+            I am currently doing a 22-week Web Development boot camp with the Swedish company Technigo.
+            ...
+            If you have any questions, do not hesitate to contact me.
+            Enjoy your day!<br>
```
Comment (JosefinRobertsson) [inline index.html]: Instead of <br> you could try a value of the property white-space: in your CSS.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+/*media queries
+
+//Tablet 
+@media (min-width: 744px) and (max-width: 1279px)
+body {
+
+}
+
+.logo {
+    display: flex;
```
Comment (JosefinRobertsson) [inline style.css]: I appreciate that you have been thinking ahead and considered what changes to display there will be on different screensizes.

**Review by JosefinRobertsson — COMMENTED** [review]
Nice touch that the links open in a new window, very user friendly. Good consideration of web accessibility, for example using correct tags such as <button> instead of styling a div, and setting an alt-text for images. Your code is easy to survey and has clear, descriptive class names, just maybe go with either uppercase or lowercase first letter in all the names? I think it would look more orderly and give an even better overview. Moving all the styling to your CSS from start would also serve the same purpose. I find the commented out part of the CSS doc inspiring, where you have made a basic layout of what to do with the styling and what media queries there will be. A smart way to structure your work and set some guideposts for yourself, that will make the work easier as you progress. Instead of finishing one section before getting to the next, you have added from the ground up and iterated over your code. This way you have made sure you have a solid base with pretty much all the content there and in the right place (if you want to you can add in the My words-section with display: none, if you plan on using it, so you don't have to build it from scratch later), and the clear structure will make the remaining styling straightforward. Great use of the time you had for this project!

**Comment by EmmaEngvall** [comment]
Have now updated the portfolio and deployed a new site:
https://project-portfolio-ee.netlify.app/

**Pull request closed without merging**